src/LinkDotNet.Blog.Web/Controller/RssFeedController.cs

```diff
@@ -105,16 +120,37 @@ private static void AddCategories(Collection<SyndicationCategory> categories, Bl
     private async Task<IEnumerable<SyndicationItem>> GetBlogPostItems(string url)
     {
         var blogPosts = await blogPostRepository.GetAllByProjectionAsync(
-            s => new BlogPostRssInfo(s.Id, s.Title, s.ShortDescription, s.UpdatedDate, s.PreviewImageUrl, s.Tags),
+            s => new BlogPostRssInfo(s.Id, s.Title, s.ShortDescription, null, s.UpdatedDate, s.PreviewImageUrl, s.Tags),
             f => f.IsPublished,
             orderBy: post => post.UpdatedDate);
         return blogPosts.Select(bp => CreateSyndicationItemFromBlogPost(url, bp));
     }
 
+    private async Task<IEnumerable<SyndicationItem>> GetBlogPostsItemsWithContent(string url, int numberOfBlogPosts)
+    {
+        var blogPosts = await blogPostRepository.GetAllByProjectionAsync(
+            s => new BlogPostRssInfo(s.Id, s.Title, null,s.Content, s.UpdatedDate, s.PreviewImageUrl, s.Tags),
```

Comment:
Super small nitpick :D 
```suggestion
            s => new BlogPostRssInfo(s.Id, s.Title, null, s.Content, s.UpdatedDate, s.PreviewImageUrl, s.Tags),
```

Reply:
Weird that the linter didn't catch it!